lib/build.py

```diff
@@ -110,6 +110,8 @@ def build_arg_get(arg):
    # Read input file.
    if (cli.file == "-" or cli.context == "-"):
       text = ch.ossafe(sys.stdin.read, "can't read stdin")
+   elif (not os.path.isdir(cli.context)):
+      ch.CONTEXT("context must be a directory: %s" % cli.context)
```

Comment:
Why not just `ch.FATAL`?

Reply:
When the `$CH_IMAGE_DEBUG` environment variable is set, `ch.FATAL` gives a stack trace. I thought this might be distracting/confusing to the user:
```
$ bin/ch-image build test/Dockerfile.00_tiny 
inferred image name: dockerfile.00_tiny
error: context must be a directory: test/Dockerfile.00_tiny
hint: build.py:115:main, ch-image:289:main
```
thoughts?

Reply:
To avoid defining a new function, I could add a keyword argument to `ch.FATAL` that suppresses the stack trace. Or we could just assume that it's not really that distracting

Reply:
I think you're right that it's distracting, but folks will have asked for that.